client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -84,7 +86,7 @@ export default function UserOperatorMultiStepForm({
     session?.user.app_role?.includes("cas") &&
     !session?.user.app_role?.includes("pending");
 
-  if (isCasInternal || !userOperatorId || formSection) {
+  if (isCasInternal || !userOperatorId) {
```

Comment:
I was testing this and the `formSection` is required or else page 2 of the multistep form shows the `UserOperatorContact` form. This will also be problematic when `CAS` users review the forms.

I think what we might be able to do is keep this logic the same and make a separate `userOperatorIdSearchParam` or something to use in page 2 of the form. I did something similar in my incoming PR here: 

https://github.com/bcgov/cas-registration/blob/e0a731e8f39a7a71faea7024ff0915d6ccd90a81/client/app/components/form/UserOperatorMultiStepForm.tsx

Reply:
Or on second thought maybe keep the param code exclusive to the `UserOperatorContactForm` ?

Reply:
Thank you for capturing this before it becomes a new bug :)